File: chat-backend/src/main/java/com/smallchat/backend/chat/framework/mongodb_adapter/MessageRepository.java
package com.smallchat.backend.chat.framework.mongodb_adapter;

import com.smallchat.backend.chat.domain.model.vo.Message;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface MessageRepository extends MongoRepository<Message, String> {
}

File: chat-backend/src/main/java/com/smallchat/backend/chat/domain/interfaces/MessageRepository.kt
package com.smallchat.backend.chat.domain.interfaces

import com.smallchat.backend.chat.domain.model.vo.Message
import org.springframework.data.mongodb.repository.MongoRepository
import org.springframework.stereotype.Repository


@Repository
@Deprecated("")
interface MessageRepository : MongoRepository<Message, String>
